packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -408,11 +409,52 @@ export interface SdkModelPropertyTypeBase extends DecoratedType {
   crossLanguageDefinitionId: string;
 }
 
+/**
+ * Options to show how to serialize a type.
+ * A type that is used in multiple operations with different wire format could have multiple options set. For example, a type could be serialized as JSON in one operation and as XML in another operation.
+ * A type that has no special serialization logic will have no options set. For example, a type that is used in a HTTP query parameter will have no serialization options set.
```

Comment:
I assume that binary payloads will also have no options set?

Reply:
yes. refined.